scripts/release/release-test.R

```diff
@@ -1008,6 +1019,11 @@ verify_ne_model_class()
 verify_ne_imp_class()
 verify_ne_lht_class()
 
+cli_alert_info("Testing dsSurvival")
+list.files(".")
+source("/cicd/scripts/release/xenon-survival.R")
```

Comment:
This is strange ... this should be a relative to `release-test.R` script include.

If it fails check sourcing from https://github.com/molgenis/molgenis-service-armadillo/blob/master/scripts/rock/rockctl/src/rockctl.R#L3-L16